src/main/java/io/github/thebusybiscuit/slimefun4/core/services/profiler/PerformanceSummary.java

```diff
@@ -65,10 +63,17 @@ public void send(@Nonnull PerformanceInspector sender) {
             int count = profiler.getBlocksOfId(entry.getKey());
             String time = NumberUtils.getAsMillis(entry.getValue());
 
-            if (count > 1) {
+            if (count > 1 && sender.getOrderType() != SummaryOrderType.AVERAGE) {
                 String average = NumberUtils.getAsMillis(entry.getValue() / count);
 
                 return entry.getKey() + " - " + count + "x (" + time + " | avg: " + average + ')';
+            } else if (count > 1 && sender.getOrderType() == SummaryOrderType.AVERAGE) {
+                String total = NumberUtils.getAsMillis(items.entrySet().stream()
+                    .filter(e -> e.getKey().equals(entry.getKey()))
+                    .mapToLong(Entry::getValue)
+                    .sum());
```

Comment:
Isn't there a helper method for this in SlimefunProfiler already?

Reply:
It's useless anyway, midnight coding haha. Fixed